backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -314,41 +314,28 @@ export class ContributionResolver {
   @Authorized([RIGHTS.ADMIN_UPDATE_CONTRIBUTION])
   @Mutation(() => AdminUpdateContribution)
   async adminUpdateContribution(
-    @Args() { id, email, amount, memo, creationDate }: AdminUpdateContributionArgs,
+    @Args() { id, amount, memo, creationDate }: AdminUpdateContributionArgs,
     @Ctx() context: Context,
   ): Promise<AdminUpdateContribution> {
     const clientTimezoneOffset = getClientTimezoneOffset(context)
-    const emailContact = await UserContact.findOne({
-      where: { email },
-      withDeleted: true,
-      relations: ['user'],
-    })
-    if (!emailContact || !emailContact.user) {
-      throw new LogError('Could not find User', email)
-    }
-    if (emailContact.deletedAt || emailContact.user.deletedAt) {
-      throw new LogError('User was deleted', email)
-    }
 
     const moderator = getUser(context)
 
     const contributionToUpdate = await DbContribution.findOne({
       where: { id, confirmedAt: IsNull(), deniedAt: IsNull() },
+      relations: ['user'],
```

Comment:
@ulfgebhardt I need the user relation only for the event here. Why did you insist on a user object?

Reply:
Furthermore. This is a contradiction to the graphql idea. I should only query the user, if I really need it. No I am forced by the event to query the user always

Reply:
`{ id: myuserId } as DBUser`